README.md

```diff
@@ -62,13 +62,28 @@ To test Helm Charts locally you will need to follow the next steps:
 13. Confirm it's gone: `helm list && kubectl get pods`
 14. Shutdown minikube: `minikube stop`
 
-To run unit tests:
+### Running Unit Tests
+To unit test the helm charts tou can follow the next steps
 
 1. Install the unittest plugin for Helm: https://github.com/quintush/helm-unittest
 2. Run the tests for each individual chart:
    * `cd charts/nexus-iq; helm unittest -3 -t junit -o test-output.xml .`
    * `cd charts/nexus-repository-manager; helm unittest -3 -t junit -o test-output.xml .`
 
+### Running Integration Tests
+You can run the integration tests for the helm charts by runnig the next commands. 
+
+Before running the integrations tests:
```

Comment:
typo: should probably be "integration tests" to match the title.

Reply:
Thanks for catching this John, I am going to fix it.